test/testdata/deployednettemplates/recipes/generate-recipe/generate_network.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+import argparse
+import simplejson
+
+import math
+import subprocess
+import os
+
+def build_network(template):
+    with open(template) as f:
+        template_dict = simplejson.load(f)
+
+    template_path = os.path.abspath(os.path.dirname(args.template))
+    script_path = os.path.dirname(__file__)
+    topology = build_topology(template_dict)
+
+    gen_dir = f"{template_path}/generated"
+    if not os.path.isdir(gen_dir):
+        os.mkdir(gen_dir)
+
+    subprocess.run(['cp', f"{script_path}/recipe.json", f"{template_path}/recipe.json"])
```

Comment:
`import shutil` probably has `cp` in a way better than subprocess

Reply:
Thanks, i switched to shutil